ui/cryptomaterial/dropdown.go

```diff
@@ -228,20 +255,55 @@ func (d *DropDown) Layout(gtx C, dropPos int, reversePos bool) D {
 		iRight = dropPos
 	}
 
+	if d.OpenedLayoutInset == nil {
+		d.OpenedLayoutInset = &layout.Inset{
+			Left:  unit.Dp(float32(iLeft)),
+			Right: unit.Dp(float32(iRight)),
+		}
+	}
```

Comment:
Don't use a pointer if nil value isn't permitted. Instead, set the default value when d is created. Callers that want to use a different value can modify as desired.